Documentation/configure-kubectl.md

```diff
@@ -42,5 +42,5 @@ X.X.X.X       kubernetes.io/hostname=X.X.X.X       Ready
 
 <div class="co-m-docs-next-step">
   <p><strong>Is kubectl working from your local machine?</strong> We're going to install an add-on with it next.</p>
-  <a href="deploy-addons.md" class="btn btn-primary btn-icon-right" data-category="Docs Next" data-event="Kubernetes: Addons">Yes, ready to deploy add-ons</a>
+  <a href="deploy-dns-add-on.md" class="btn btn-primary btn-icon-right" data-category="Docs Next" data-event="Kubernetes: DNS add-on">Yes, ready to deploy DNS add-on</a>
```

Comment:
File location and data-event need to stay the same. The link text is okay to change.